icons/square-equal.json

```
@@ -0,0 +1,11 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "calculate",
+    "maths"
```

```suggestion
    "calculate"
```

Just the category will suffice in the future I think.

---

So does the search function treat `categories` and `tags` (and `aliases`, for that matter) the same then?

---

Not sure about right now, but it certainly should and it will in the future (as well as display both in the list).